backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -507,13 +505,14 @@ def _create_deployment(
                     name="workspace", mount_path="/var/log/promtail"
                 )
             )
-
-        volumes.append(
-            client.V1Volume(
-                name="prom-config",
-                config_map=client.V1ConfigMapVolumeSource(name=name),
+        if loki_enabled:
+            volumes.append(
+                client.V1Volume(
+                    name="prom-config",
+                    config_map=client.V1ConfigMapVolumeSource(name=name),
+                )
             )
-        )
+
         promtail_volume_mounts.append(
```

Comment:
This shouldn't be executed if loki_enabled is false, right?

Reply:
I refactored this part since there was also an issue that we do not capture read only session logs. So now there is  a more clear separation and everything promtail related is now placed inside the `if loki_enabled` block.